package.json

```diff
@@ -93,19 +93,19 @@
     "mocha": "^5.2.0",
     "mocha-junit-reporter": "^1.18.0",
     "mocha-multi-reporters": "^1.1.7",
-    "node-sass": "^4.7.2",
+    "node-sass": "^4.11.0",
```

Comment:
We need to be careful about the version of Node that ships in Electron (which will be Node 10.x in Electron 3) versus the Node version used as part of the toolchain (still Node 8 until we resolve #5876). This is reflected in how we split up the `package.json` dependencies:

 - `package.json` - toolchain for building Desktop
 - `app/package.json` - libraries used inside Desktop

My working assumption here is that anything related to Electron 3 should only require changing versions in `app/package.json` (aside from `electron-*` dependencies). Can we confirm we're not trying to upgrade _everything_ to Node 10 as part of this PR?